pkg/kubelet/kuberuntime/kuberuntime_sandbox.go

```diff
@@ -141,14 +140,11 @@ func (m *kubeGenericRuntimeManager) generatePodSandboxConfig(pod *v1.Pod, attemp
 		return nil, err
 	}
 	podSandboxConfig.Linux = lc
```

Comment:
do we need to set linux context on Windows node? Looks counterintuitive

Reply:
It seems so, according to the other comment I've left.